docs/settings/apm-settings.asciidoc

```diff
@@ -21,6 +21,8 @@ xpack.apm.ui.enabled:: Set to `false` to hide the APM plugin {kib} from the menu
 
 xpack.apm.ui.transactionGroupBucketSize:: Number of top transaction groups displayed in APM plugin in Kibana. Defaults to `100`.
 
+xpack.apm.ui.maxTraceItems:: Max number of child items displayed when viewing trace details. Defaults to `1000`.
```

Comment:
It used to be required to add new options to the docker image (or they wouldn't be available for users using the official Kibana Docker image).
Here' s an example of some settings I added to the whitelist: https://github.com/elastic/kibana-docker/pull/129

However, that repo has been archived and it's been merged with the normal kibana repo afaik. Would be good to find out if this step is still required.